packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -77,6 +87,16 @@ const WRAPPER_PATH = 'webxdc-wrapper.45870014933640136498.html'
 
 const BOUNDS_UI_CONFIG_PREFIX = 'ui.desktop.webxdcBounds'
 
+const DEFAULT_BOUNDS_WEBXDC = {
+  width: 375,
+  height: 667,
+}
+
+const DEFAULT_BOUNDS_MAP = {
+  width: 1000,
+  height: 800,
+}
+
```

Comment:
What about doing something like this instead?

![image](https://github.com/user-attachments/assets/e06dd5e6-cd0b-4230-9a06-bde57b3929cc)